frontend/src/components/Faq/FaqTab.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useRef, useState } from 'react';
+import { observer } from 'mobx-react';
+import sanitizeHtml from 'sanitize-html';
+import classnames from 'classnames';
+
+import {
+  Row,
+  Col,
+  SideMenu,
+  SideMenuLink,
+  Accordion,
+  AccordionItem,
+  Title,
+} from '@dataesr/react-dsfr';
+
+import FaqProcess from 'components/Faq/FaqProcess';
+
+import { useStore } from 'stores/';
+import faq from 'services/faq/faq';
+import items from 'services/faq/items';
+
+import styles from './faqTab.cssmodule.scss';
+
+const FaqTab = ({ type }) => {
+  const refs = items[type].sections.map(() => useRef(null));
+  const { commonStore: { config } } = useStore();
+  const [activeSection, setActiveSection] = useState();
+
+  return (
+    <div data-test-id={`tabpanel-${type}`}>
+      <FaqProcess
+        label={items[type].label}
+        links={items[type].links}
+      />
+      <Row spacing="my-3w">
+        <SideMenu
+          buttonLabel="Dans cette rubrique"
+          className={classnames(styles.menu, 'fr-sidemenu--sticky fr-col-md-4 fr-col-sm-12')}
+        >
+          {items[type].sections.map((section, index) => (
+            <SideMenuLink
+              key={section.title}
+              onClick={() => {
+                refs[index].current.scrollIntoView();
+                setActiveSection(section);
+              }}
+              className={
+                      activeSection && activeSection.title === section.title ? 'fr-sidemenu__item--active' : ''
+                    }
+            >
+              {section.title}
+            </SideMenuLink>
+          ))}
+        </SideMenu>
+        <Col n="md-8 sm-12" className={styles.sections}>
+          {items[type].sections.map((section, index) => (
+            <div ref={refs[index]} key={section.name} id={section.id ? section.id : null}>
+              <Title as="h2" look="h4">{section.title}</Title>
```

Comment:
pas d'interet a le forcer en h2 ? 
```suggestion
              <Title as="h4" look="h4">{section.title}</Title>
```

Reply:
Il me semble que c'est une règle liée à l'accessibilité (je ne l'ai pas trouvé dans le référentiel RGAA cela dit :thinking:), d'avoir un ordre de titre qui se suit h1, h2, etc. Je pense que c'est pour cette raison qu'il y a `as` et `look`.